testdata/e2e/README.md

```diff
@@ -0,0 +1,61 @@
+E2E tests
```

Comment:
This is great.  Add more test coverage though?

Reply:
Yes, I'll make sure both missing limits and too-expensive expressions get coverage. I think having that and merging #1 (plus resolving any outstanding feedback) should be good enough to make this a non-draft PR.

Reply:
(marking this as resolved now that JSON and regular output both have E2E tests)